tap_github/authenticator.py

```diff
@@ -1,11 +1,14 @@
 """Classes to assist in authenticating to the GitHub API."""
 
 import logging
+import random
+import time
 from datetime import datetime
 from os import environ
 from random import choice, shuffle
 from typing import Any, Dict, List, Optional
 
+import jwt
```

Comment:
cannot, the singer-sdk is stuck at 1.7.1

Reply:
@ericboucher @laurentS are guys willing to submit an MR to update this dependency in the SDK? I think it's a safe one since we only use [`jwt.encode`](https://gitlab.com/meltano/sdk/-/blob/11638bbd4cf20e9c0cdd43884c705c79e6c5aa01/singer_sdk/authenticators.py#L505) and that method hasn't changed its signature 🙂

Reply:
https://gitlab.com/meltano/sdk/-/merge_requests/281